core/services/chainlink/config_test.go

```diff
@@ -0,0 +1,769 @@
+
+				RPCDefaultBatchSize:    ptr[uint32](17),
+				TxReaperInterval:       &minute,
+				TxReaperThreshold:      &minute,
+				TxResendAfterThreshold: &hour,
+				UseForwarders:          ptr(true),
+			},
+			Nodes: []evmtypes.TOMLNode{
+				{
+					Name:    ptr("foo"),
+					HTTPURL: mustURL("https://foo.web"),
+					WSURL:   mustURL("wss://web.socket/test"),
+				},
+				{
+					Name:    ptr("bar"),
+					HTTPURL: mustURL("https://bar.com"),
+					WSURL:   mustURL("wss://web.socket/test"),
+				},
+				{
+					Name:     ptr("broadcast"),
+					HTTPURL:  mustURL("http://broadcast.mirror"),
+					SendOnly: ptr(true),
+				},
+			}},
+	}
+	full.Solana = []SolanaConfig{
+		{
+			ChainID: "mainnet",
+			Enabled: ptr(false),
+			TOMLChain: solana.TOMLChain{
+				BalancePollPeriod:   models.MustNewDuration(time.Minute),
+				ConfirmPollPeriod:   models.MustNewDuration(time.Second),
+				OCR2CachePollPeriod: models.MustNewDuration(time.Minute),
+				OCR2CacheTTL:        models.MustNewDuration(time.Hour),
+				TxTimeout:           models.MustNewDuration(time.Hour),
+				TxRetryTimeout:      models.MustNewDuration(time.Minute),
+				TxConfirmTimeout:    models.MustNewDuration(time.Second),
+				SkipPreflight:       ptr(true),
+				Commitment:          ptr("banana"),
+				MaxRetries:          ptr[int64](7),
+			},
+			Nodes: []solana.TOMLNode{
+				{Name: "primary", URL: mustURL("http://solana.web")},
+				{Name: "foo", URL: mustURL("http://solana.foo")},
+				{Name: "bar", URL: mustURL("http://solana.bar")},
+			},
+		},
+	}
+	full.Terra = []TerraConfig{
+		{
+			ChainID: "Bombay-12",
+			TOMLChain: terra.TOMLChain{
+				BlockRate:             models.MustNewDuration(time.Minute),
+				BlocksUntilTxTimeout:  ptr[int64](12),
+				ConfirmPollPeriod:     models.MustNewDuration(time.Second),
+				FallbackGasPriceULuna: mustDecimal("0.001"),
+				FCDURL:                mustURL("http://terra.com"),
+				GasLimitMultiplier:    mustDecimal("1.2"),
+				MaxMsgsPerBatch:       ptr[int64](17),
+				OCR2CachePollPeriod:   models.MustNewDuration(time.Minute),
+				OCR2CacheTTL:          models.MustNewDuration(time.Hour),
+				TxMsgTimeout:          models.MustNewDuration(time.Second),
+			},
+			Nodes: []terra.TOMLNode{
+				{Name: "primary", TendermintURL: mustURL("http://tender.mint")},
+				{Name: "foo", TendermintURL: mustURL("http://foo.url")},
+				{Name: "bar", TendermintURL: mustURL("http://bar.web")},
+			},
+		},
+	}
+
+	for _, tt := range []struct {
+		name   string
+		config Config
+		exp    string
+	}{
+		{"empty", Config{}, ``},
+		{"global", global, `Dev = true
+ExplorerURL = 'http://explorer.url'
+InsecureFastScrypt = true
+ReaperExpiration = '168h0m0s'
+RootDir = 'test/root/dir'
+ShutdownGracePeriod = '10s'
+FeatureFeedsManager = true
+FeatureUICSAKeys = true
+FeatureLogPoller = true
+FMDefaultTransactionQueueDepth = 100
+FMSimulateTransactions = true
+FeatureOffchainReporting2 = true
+FeatureOffchainReporting = true
```

Comment:
@samsondav Wdyt about swapping these out for implicit `true` if the `[OCR]`/`[OCR2]` table is present, but also allowing explicit `false` to disable?
```toml
[OCR]
Enabled = false

[OCR2]
Enabled = false
```